ansible_collections/arista/avd/docs/installation/collection-installation.md

```diff
@@ -1,14 +1,19 @@
-# Collection Installation
+# Collection installation
 
-These instructions are for regular users to install via Ansible Galaxy. To setup a development environment use [these](../contribution/setup-environment.md) instructions.
+## Installation workflow
 
-**arista.avd** can also be consumed using the ["AVD All-in-one" container](https://github.com/arista-netdevops-community/avd-all-in-one-container).
+- Python **3.8** or later
```

Comment:
Should say "Install Python **3.8** or later" to follow the style of the later points.

Reply:
same for ansible.

Reply:
agreed